### DISCUSSION TITLE
Add coordinates to URL parameters

### DISCUSSION THREAD

**Discussion by alex-oser** [lowlighter/gracidea#63] [Ideas]
It would be nice to be able to link someone to a specific location on the map by pulling coordinates from the URL.

Playing around with it locally, it seems pretty straightforward. This pulls the params from the URL and keeps the current default. Next part (and probably the more complex part), need to make it so that moving around the map updates the URL params.

```js
const params = new URLSearchParams(window.location.search)
const x = params.get("x") || 329;
const y = params.get("y") || -924;
```

**Comment by lowlighter** [comment]
Hi!

Yes, it used to be a feature of the v1 that got removed in v2 but eventually it should come back.

The old version was also using `history.pushState` (or similar API, I don't remember the exact name) so the URL was refreshed in real time so you could share the exact place you were 🙂 

Thanks for your suggestion!
